sentry/sentry.conf.example.py

```diff
@@ -141,25 +141,6 @@ def get_internal_network():
     }
 }
 
-#########
-# Queue #
-#########
-
-# See https://develop.sentry.dev/services/queue/ for more
-# information on configuring your queue broker and workers. Sentry relies
-# on a Python framework called Celery to manage queues.
-
-rabbitmq_host = None
```

Comment:
@aldy505 @markstory would this require a migration for existing setups or would it just be a useless config definition for them until they delete?

Reply:
Useless config definition I believe